Documents/Installation-with-Conan.md

```diff
@@ -0,0 +1,57 @@
+## Guide to Install and Setup nmos-cpp using Conan Package Manager: 
+
+1. Install conan:
+`pip install --upgrade conan>=2.0.5`
+
+    If conan.exe is not in PATH you will get warning: 
+	 
+
+	> WARNING: The script conan.exe is installed in 'C:\Users\%USER%\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\Scripts' which is not on PATH.
+	  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
+
+
+   Then the shown path needs to be added manually. After maunally adding the PATH, Restart the command line and run `conan --help` to ensure Conan is added to path successfully.
+
+3. Detect Profile:
+	Run `conan profile detect`. \
+  This will create a conan profile with values based on your dev setup. Ideally the profile should be similar to these:
+	
+   For windows:
+	```{txt}
+	[settings]
+	arch=x86_64
+	build_type=Release
+	compiler=msvc
+	compiler.cppstd=14
+	compiler.runtime=dynamic
+	compiler.runtime_type=Release
+	compiler.version=193
+	os=Windows
+	```
+	For Linux:
+	```{txt}
+	[settings]
+	arch=x86_64
+	build_type=Release
+	compiler=gcc
+	compiler.cppstd=17
+	compiler.version=11
+	os=Linux
+	```
```

Comment:
- spaces not tabs (also applies in other places)
- `ini` has nicer formatting for this sort of config file than `txt`
- removed the Windows `compiler.runtime_type` because I read somewhere (sorry, no reference) that it causes trouble?

~~~suggestion
   Windows
   ```ini
   [settings]
   arch=x86_64
   build_type=Release
   compiler=msvc
   compiler.cppstd=14
   compiler.runtime=dynamic
   compiler.version=193
   os=Windows
   ```
   Linux
   ```ini
   [settings]
   arch=x86_64
   build_type=Release
   compiler=gcc
   compiler.cppstd=17
   compiler.version=11
   os=Linux
   ```
~~~